DistantWorlds2.ModLoader.ModManager/ModManager.cs

```diff
@@ -30,16 +31,15 @@ public class ModManager : IModManager {
 
   private string? _modsDir;
 
+  private byte[]? _checksum;
```

Comment:
```suggestion
  // Game data checksum
  private byte[]? _checksum;
```
or ...
```suggestion
  private byte[]? _gameDataChecksum;
```
(would require refactoring usages)

N.B.